dd-java-agent/instrumentation/jdbc/src/main/java/datadog/trace/instrumentation/jdbc/JDBCDecorator.java

```diff
@@ -45,7 +45,14 @@ protected String dbUser(final JDBCMaps.DBInfo info) {
 
   @Override
   protected String dbInstance(final JDBCMaps.DBInfo info) {
-    return info.getUrl();
+    String dbName = null;
+    String urlInfo = info.getUrl();
+    String[] urlArray = urlInfo.split("/");
+    if (urlArray.length > 1) {
```

Comment:
Is this condition necessary?

Reply:
> Can you add truncation to https://github.com/signalfx/signalfx-java-tracing/blob/master/dd-java-agent/agent-tooling/src/main/java/datadog/trace/agent/decorator/DatabaseClientDecorator.java#L39?

You want me to truncate the `DB_STATEMENT`? I thought we always allowed the full statement?

Reply:
> Is this condition necessary?

If `urlInfo` is empty or contains no slashes, then I think it will be helpful to have that check.